templates/environment.j2

```diff
@@ -0,0 +1,3 @@
+{% for e in django_venv_vars %}
+{{ e }}={{ django_venv_vars[e] }}
+{% endfor %}
```

Comment:
Since this is a dict, we can use `.iteritems()` in the loop and use key/value unpacking. Will update as part of review.